cirq-core/cirq/sim/act_on_state_vector_args.py

```diff
@@ -177,7 +467,7 @@ def _act_on_fallback_(
             _strat_act_on_state_vector_from_channel,
         ]
         if allow_decompose:
-            strats.append(strat_act_on_from_apply_decompose)
+            strats.append(strat_act_on_from_apply_decompose)  # type: ignore
```

Comment:
Prefer assigning a fitting type to strats (e.g. `strats: Callable[...] = [...]`) if possible.